### PR TITLE
Rename reorder_dwi

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-835/1/

File: scripts/scil_dwi_reorder_philips.py [hunk 1]
```diff
@@ -55,9 +63,21 @@ def main():
                         args.out_basename + '.bval',
                         args.out_basename + '.bvec']
 
-    assert_inputs_exist(parser, required_args)
+    assert_inputs_exist(parser, required_args, args.json)
     assert_outputs_exist(parser, args, output_filenames)
 
+    if args.json and args.force:
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:69]: Remove the `and args.force`.

File: scripts/scil_dwi_reorder_philips.py [hunk 2]
```diff
@@ -55,9 +63,21 @@ def main():
...
+        with open(args.json) as curr_json:
+            dwi_json = json.load(curr_json)
+        if 'SoftwareVersions' in dwi_json.keys():
+
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:73]: Remove blank line.

File: scripts/scil_dwi_reorder_philips.py [hunk 3]
```diff
@@ -55,9 +63,21 @@ def main():
...
+            curr_version = dwi_json['SoftwareVersions']
+            curr_version = curr_version.replace('\\', ' ').replace('_', ' ')
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:75]: Replace this line by `curr_version = curr_version.split('_')[0]`.

File: scripts/scil_dwi_reorder_philips.py [hunk 4]
```diff
@@ -55,9 +63,21 @@ def main():
...
+            if version.parse(SOFTWARE_VERSION_MIN) <= version.parse(curr_version):
+                logging.error('There is no need for reording since your'
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:77]: I would add a space after `your` and remove the space before `dwi` :P

File: scripts/scil_dwi_reorder_philips.py [hunk 5]
```diff
@@ -55,9 +63,21 @@ def main():
...
+                              ' dwi comes from a philips machine '
+                              'version {}'.format(curr_version))
+
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:80]: Would it really continue anyway?

File: scripts/scil_dwi_reorder_philips.py (hunk 3 above)
Comment (karanphil) [reply scripts/scil_dwi_reorder_philips.py:75]: Ok then put 
`curr_version = curr_version.replace('\\', '_ ')`
`curr_version = curr_version.split('_')[0]`

**Comment by karanphil** [comment]
Also, I would specify in the doc at the top that this script is not needed for version R5.6 and higher.

File: scripts/scil_dwi_reorder_philips.py [hunk 6]
```diff
@@ -55,9 +65,24 @@ def main():
...
+        with open(args.json) as curr_json:
+            dwi_json = json.load(curr_json)
+        if 'SoftwareVersions' in dwi_json.keys():
+            curr_version = dwi_json['SoftwareVersions']
+            curr_version = curr_version.replace('\\',
+                                                ' ').replace('_',
+                                                             ' ').split()[0]
+            if version.parse(SOFTWARE_VERSION_MIN) <= version.parse(curr_version):
+                sys.exit('ERROR: There is no need for reording since your '
+                         'dwi comes from a philips machine with '
```
Comment (karanphil) [inline scripts/scil_dwi_reorder_philips.py:81]: philips -> Philips

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**